### ISSUE TITLE
Change sub question 1 -'What are the common trends among the most commonly used passwords?' and refactor analysis on section

### ISSUE THREAD

**Issue by dilinie-seimon** [etc5521-2020/assignment-1-numbat#11]
I feel the first subquestion 'What are the common trends among the most commonly used passwords?' is unclear.
The analysis points out the most common passwords used based on the popularity rank, the password categories with the most number of passwords, distribution of passwords based on their lengths and a summary of lengths of different password categories.

I suggest we remove unnecessary plots and text and change  the question to 'What are the most common passwords and password categories based on their popularity, usage, and length?'

**Comment by jluo0015** [comment]
Yes, by further reviewing, section3.1 is not clear enough, we need to remove unnecessary parts and give some more clear explanation.
